src/containers/settings/EditServiceScreen.js

```diff
@@ -208,9 +208,7 @@ class EditServiceScreen extends Component {
           label: intl.formatMessage(messages.trapLinkClicks),
           // THIS OVERRIDES THE CURRENT EXISTING RECIPES SETTINGS IF THE GLOBAL SETTING IS SET TO TRUE. 
           // TODO: SOLUTION FOR WHEN GLOBAL SETTING IS TRUE AND THE USER WANTS TO SET A SPECIFIC SERVICE TO OFF.
```

Comment:
or - it could be done the other way around - default (at the code-level) being overridden by the global which in turn is overridden by the service-specific one. That would be the inverse of your implementation. but, to me, that makes sense

Reply:
I have removed the global equivalent - since it introduces a lot of complication like which setting did the user change last. Coupled with the defaulting of this preference if unset, there are too many combinations possible and no coding could resolve that in a clean manner. Thus the removal of the global preference.